qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -444,10 +444,316 @@ def particle_number(orbitals, mapping="jordan_wigner", wires=None):
+        \hat{c}_{\gamma\uparrow} \hat{c}_{\delta\downarrow} + \hat{c}_{\alpha\downarrow}^\dagger
+        \hat{c}_{\beta\downarrow}^\dagger \hat{c}_{\gamma\downarrow} \hat{c}_{\delta\downarrow}~].
+
+    In the equation above the indices :math:`\alpha, \beta, \gamma, \delta` run over the basis
+    of spatial orbitals :math:`\phi_\alpha(r)`. Since the operator :math:`v` acts only on the
+    spatial coordinates the spin quantum numbers are indicated explicitly with the up/down arrows.
+    The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}` are the particle creation and
+    annihilation operators, respectively, and
+    :math:`\langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle` denotes the
+    matrix elements of the operator :math:`\hat{v}`
+
+    .. math::
+
+        \langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle =
+        \int dr_1 \int dr_2 ~ \phi_\alpha^*(r_1) \phi_\beta^*(r_2) ~\hat{v}(r_1, r_2)~
+        \phi_\gamma(r_2) \phi_\delta(r_1).
+
+    If an active space is defined (see :func:`~.active_space`), the summation indices
+    run over the active orbitals and the contribution due to core orbitals, is computed as
+    :math:`V_\mathrm{core} = 2 \sum_{\alpha, \beta \in \mathrm{core}}
+    \langle \alpha, \beta \vert \hat{v} \vert \alpha, \beta \rangle`.
+
+    Args:
+        matrix_elements (array[float]): 4D NumPy array with the matrix elements
+            :math:`\langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle`
+        core (list): indices of core orbitals, i.e., the orbitals that are
+            not correlated in the many-body wave function
+        active (list): indices of active orbitals, i.e., the orbitals used to
+            build the correlated many-body wave function
+        cutoff (float): Cutoff value for including matrix elements. The
+            matrix elements with absolute value less than ``cutoff`` are neglected.
+
+    Returns:
+        tuple: Table with indices and matrix elements of the two-particle operator
+        and the contribution due to core orbitals. The returned table is a 2D Numpy
+        array where each row contains five elements, the *spin*-orbital indices
+        :math:`\alpha, \beta, \gamma, \delta` and the matrix element
+        :math:`\langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle`.
+
+    **Example**
+
+    >>> matrix_elements = np.array([[[[ 6.82389533e-01, -1.45716772e-16],
+    ...                               [-2.77555756e-17,  1.79000576e-01]],
+    ...                              [[-2.77555756e-17,  1.79000576e-16],
+    ...                               [ 6.70732778e-01, 0.00000000e+00]]],
+    ...                             [[[-1.45716772e-16,  6.70732778e-16],
+    ...                               [ 1.79000576e-01, -8.32667268e-17]],
+    ...                              [[ 1.79000576e-16, -8.32667268e-17],
+    ...                               [ 0.00000000e+00,  7.05105632e-01]]]])
+
```

Comment:
```suggestion
```
This suggestion goes from
![image](https://user-images.githubusercontent.com/49409390/94070282-eae9e780-fdbf-11ea-9795-9a2b926e5356.png)

to
![image](https://user-images.githubusercontent.com/49409390/94070357-0b19a680-fdc0-11ea-8696-8b9c8716b7ba.png)

It's mainly a matter of taste!